Source/Editor/DualityEditor/DualityEditorApp.cs

```diff
@@ -407,146 +401,6 @@ public static IEnumerable<IEditorAction> GetEditorActions(Type subjectType, IEnu
 			}
 		}
 
-		public static void SaveUserData()
-		{
-			Logs.Editor.Write("Saving user data...");
-			Logs.Editor.PushIndent();
-
-			using (FileStream str = File.Create(UserDataFile))
-			{
-				Encoding encoding = Encoding.Default;
-				using (StreamWriter writer = new StreamWriter(str.NonClosing()))
-				{
-					encoding = writer.Encoding;
-
-					XDocument xmlDoc = new XDocument();
-					XElement rootElement = new XElement("UserData");
-					{
-						XElement editorAppElement = new XElement("EditorApp");
-						{
-							editorAppElement.SetElementValue("Backups", backupsEnabled);
-							editorAppElement.SetElementValue("Autosaves", autosaveFrequency);
-							editorAppElement.SetElementValue("LauncherPath", launcherApp);
-							editorAppElement.SetElementValue("FirstSession", false);
-							editorAppElement.SetElementValue("ActiveDocumentIndex", mainForm.ActiveDocumentIndex);
-							editorAppElement.SetElementValue("LastOpenScene", lastOpenScene.Path);
-							editorAppElement.SetElementValue("StartWithLastScene", startWithLastScene);							
```

Comment:
I just noticed that this PR replaces the old manual serialization of all the `DualityEditorApp` user settings with the new user settings class, but actually never accesses its values. The editor still relies on the internal fields and properties, but they're never serialized or deserialized. We should fix this before merging.